bin/authenticate

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
FWIW I don't think this type of boilerplate is really necessary for this type of project, see mozilla/gcp-ingestion#777